bitcoin/esplora/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:14-alpine3.13 AS build
+
+RUN apk add --update --no-cache \
+    git \
+    bash
+
+WORKDIR /workdir
+
+RUN git clone --no-checkout https://github.com/Blockstream/esplora.git
+
+WORKDIR esplora
+
+RUN git checkout f6da494a9a0080ae2ae13842bcf53b259e3a3360
+
+RUN npm install
+
+ENV CORS_ALLOW="*"
+
+ARG API_URL
+ENV API_URL=$API_URL
+
+RUN npm run dist
+
+FROM nginx:1.17-alpine
```

Comment:
Why 1.17 instead of 1.20? Does this correspond to the same alpine version from the build layer?

Reply:
Copy-paste from another place. I forgot to update it to the latest. Will do!